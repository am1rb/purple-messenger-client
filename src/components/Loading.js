// @flow

import React from 'react'

import styles from './Loading.css'

type Props = {};


export default class Loading extends React.PureComponent<Props>
{
	render()
	{
		return (
			<div className={styles.loading}>
				loading...
			</div>
		)
	}
}