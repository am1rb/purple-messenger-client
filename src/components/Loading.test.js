// @flow

import React from 'react'
import renderer from 'react-test-renderer'

import Loading from './Loading'


describe("<Loading /> component tests", () => {

	it("Should not regress", () => {

		const component = renderer.create(
			<Loading />
		)

		expect(component.toJSON()).toMatchSnapshot()

	})

})