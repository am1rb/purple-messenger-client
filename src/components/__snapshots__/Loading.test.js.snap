// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Loading /> component tests Should not regress 1`] = `
<div
  className={undefined}
>
  loading...
</div>
`;
